edit-post/components/layout/index.js

```diff
@@ -111,6 +107,7 @@ function Layout( {
 					openedGeneralSidebar={ openedGeneralSidebar } />
 			}
 			<Popover.Slot />
+			<PluginFills />
```

Comment:
This component might be renamed to `PluginArea` or something like this to better reflect its role.